### PR TITLE
[feature] Added hoppers to Stashfinder

### PR DESCRIPTION
**Pull request by widdwecat**
I hope this works

Hey, it's me again. Again, I am just adding something simple.

### CODE REVIEW

File: src/main/java/me/zeroeightsix/kami/module/modules/misc/StashFinder.kt [hunk 1]
```diff
@@ -33,6 +33,10 @@ object StashFinder : Module() {
 
     private val logDispensers = register(Settings.b("Dispensers", false))
     private val dispenserDensity = register(Settings.integerBuilder("MinDispensers").withMinimum(1).withMaximum(20).withValue(5).withVisibility { logDispensers.value }.build())
+        
+    private val logHoppers = register(Settings.b("Hoppers", false))
+    private val hopperDensity = register(Settings.integerBuilder("MinHoppers").withMinimum(1).withMaximum(20).withValue(5).withVisibility { logDispensers.value }.build())
```
Comment (l1ving) [inline src/main/java/me/zeroeightsix/kami/module/modules/misc/StashFinder.kt:38]: you forgot to change the visibility

**Review by l1ving — CHANGES_REQUESTED** [review]

File: src/main/java/me/zeroeightsix/kami/module/modules/misc/StashFinder.kt (hunk 1 above)
Comment (widdwecat) [reply src/main/java/me/zeroeightsix/kami/module/modules/misc/StashFinder.kt:38]: sorry, I'll change that now.

File: src/main/java/me/zeroeightsix/kami/module/modules/misc/StashFinder.kt [hunk 2]
```diff
@@ -62,7 +66,7 @@ object StashFinder : Module() {
         }
 
         override fun toString(): String {
-            return "($chests chests, $shulkers shulkers, $droppers droppers, $dispensers dispensers)"
+            return "($chests chests, $shulkers shulkers, $droppers droppers, $dispensers dispensers, $hoppers hoppers)"
```
Comment (l1ving) [inline src/main/java/me/zeroeightsix/kami/module/modules/misc/StashFinder.kt:69]: you forgot to make a variable for hoppers

File: src/main/java/me/zeroeightsix/kami/module/modules/misc/StashFinder.kt [hunk 3]
```diff
@@ -84,14 +88,14 @@ object StashFinder : Module() {
         val chunkStats = chunkData.getOrPut(chunk, { ChunkStats() })
 
         chunkStats.add(tileEntity)
-        if (chunkStats.chests >= chestDensity.value || chunkStats.shulkers >= shulkerDensity.value || chunkStats.droppers >= dropperDensity.value || chunkStats.dispensers >= dispenserDensity.value) {
+        if (chunkStats.chests >= chestDensity.value || chunkStats.shulkers >= shulkerDensity.value || chunkStats.droppers >= dropperDensity.value || chunkStats.dispensers >= dispenserDensity.value || chunkStats.hoppers >= hopperDensity.value) {
```
Comment (l1ving) [inline src/main/java/me/zeroeightsix/kami/module/modules/misc/StashFinder.kt:91]: see above

File: src/main/java/me/zeroeightsix/kami/module/modules/misc/StashFinder.kt [hunk 4]
```diff
@@ -33,6 +33,10 @@ object StashFinder : Module() {
 
     private val logDispensers = register(Settings.b("Dispensers", false))
     private val dispenserDensity = register(Settings.integerBuilder("MinDispensers").withMinimum(1).withMaximum(20).withValue(5).withVisibility { logDispensers.value }.build())
+        
+    private val logHoppers = register(Settings.b("Hoppers", false))
+    private val hopperDensity = register(Settings.integerBuilder("MinHoppers").withMinimum(1).withMaximum(20).withValue(5).withVisibility { logHoppers.value }.build())
+
 
     private data class ChunkStats(var chests: Int = 0, var shulkers: Int = 0, var droppers: Int = 0, var dispensers: Int = 0, var hot: Boolean = false) {
```
Comment (l1ving) [inline src/main/java/me/zeroeightsix/kami/module/modules/misc/StashFinder.kt:41]: you're missing hoppers

**Review by l1ving — CHANGES_REQUESTED** [review]
You're also missing hoppers inside `add()`

File: src/main/java/me/zeroeightsix/kami/module/modules/misc/StashFinder.kt (hunk 4 above)
Comment (widdwecat) [reply src/main/java/me/zeroeightsix/kami/module/modules/misc/StashFinder.kt]: I just added it

**Review by l1ving — APPROVED** [review]
Thanks

**Pull request merged**